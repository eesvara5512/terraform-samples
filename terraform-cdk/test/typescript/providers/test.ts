/**
 * Testing interaction with Terraform Cloud
 *
 * @group typescript
 */

import { TestDriver } from "../../test-helper";

describe("full integration test", () => {
  let driver: TestDriver;

  beforeAll(async () => {
    driver = new TestDriver(__dirname);
    await driver.setupTypescriptProject();
  });

  test("build providers", () => {
    expect(driver.deploy()).toMatchSnapshot();
  });
});
