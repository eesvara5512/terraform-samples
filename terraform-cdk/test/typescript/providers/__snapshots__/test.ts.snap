// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`full integration test build providers 1`] = `
"Deploying Stack: hello-deploy
Resources
 ✔ NULL_RESOURCE       test                null_resource.test


Summary: 1 created, 0 updated, 0 destroyed.
"
`;
